src/overlays/actors/ovl_En_Ru2/z_en_ru2.c

```diff
@@ -217,7 +247,9 @@ void func_80AF2868(EnRu2* this, PlayState* play, u32 cueChannel) {
     }
 }
 
-void func_80AF28E8(EnRu2* this, AnimationHeader* animation, u8 arg2, f32 morphFrames, s32 arg4) {
+/* Changes the animation for Ruto's actor. The direction argument decides whether to play the animation
+   forwards (if 0) or backwards (otherwise). */
```

Comment:
did you mean to rename `arg4` to `direction` here?

Reply:
Oops, yes. And there are other args/vars to rename too.